test/functional/api/v1/searches_controller_test.rb

```diff
@@ -59,7 +59,7 @@ def self.should_respond_to(format)
     end
 
     should "default to first page when page is not a number" do
-      get :show, query: "match", page: "foo", format: :json
+      get :show, query: "match", page: {}, format: :json
```

Comment:
`"foo".to_i` and `{}.to_i` are not same. Please add your test instead of modifying existing one.